File: 100-exercicios-js/lacos-de-repeticao/tabuada.js
/***
 * 
 * 
 * 50. crie um programa que solicite ao usuario 
 * um numero e exiba a tabuada desse numero
 * 
 * 
 */
const numero = prompt("digite um numero : ");

console.log(`Tabuada do ${numero}`);

for(let i = 1; i <= 10; i++) {
  const resultado = numero * i;
  console.log(`${numero} X ${i} = ${resultado}`)
}

File: 200-exercicios-java/src/main/java/br/com/capitulo4/Tabuada.java
package br.com.capitulo4;

import java.util.Scanner;

/*
 * 
 * Crie um programa que exiba a tabuada de um número fornecido pelo usuário, de 1 a 10.
 * 
 * 
 * */
public class Tabuada {
	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		
		System.out.println("digite um numero para a tabuada : ");
		int numero = scanner.nextInt();
		
		for(int i = 1;i <= 10;i++) {
			System.out.println(numero+" x "+i+" = "+(numero*i));
		}
		
		scanner.close();
	}
}
